src/app/pages/communication/communication.page.ts

```diff
@@ -216,10 +224,14 @@ export class CommunicationPage extends PracticeableBasePageComponent implements
   }
 
   dispatchCandidateChoseNewEmail(communicationEmail: string): void {
-    this.setCommunicationType(CommunicationPage.email, CommunicationPage.updatedEmail);
-    this.store$.dispatch(communicationPreferencesActions.CandidateChoseEmailAsCommunicationPreference(
-      communicationEmail, CommunicationPage.email,
-    ));
+    console.log('this.candidateProvidedEmail', this.candidateProvidedEmail);
+    console.log('this.communicationEmail', this.communicationEmail)
+    if(this.userAction !== userActions.provided) {
```

Comment:
this is the bit that needs tweaking.  The obvious is this.candidateProvidedEmail !== this.communicationEmail which gives needed functionality, but doesnt update the state